Submodule qtbase 01272c09..0a166b7a:
diff --git a/qtbase/mkspecs/features/qt_module.prf b/qtbase/mkspecs/features/qt_module.prf
index ee7de22059..6bd7e36d9b 100644
--- a/qtbase/mkspecs/features/qt_module.prf
+++ b/qtbase/mkspecs/features/qt_module.prf
@@ -210,54 +210,6 @@ equals(QT_ARCH, i386):contains(QT_CPU_FEATURES.$$QT_ARCH, sse2):compiler_support

 android: CONFIG += qt_android_deps no_linker_version_script

-!header_module:unix:!isEmpty(QMAKE_LFLAGS_VERSION_SCRIPT):!no_linker_version_script:!static {
-    verscript = $${TARGET}.version
-    QMAKE_LFLAGS += $${QMAKE_LFLAGS_VERSION_SCRIPT}$$verscript
-
-    internal_module {
-        verscript_content = "Qt_$${QT_MAJOR_VERSION}_PRIVATE_API { *; };"
-    } else {
-        verscript_content = "Qt_$${QT_MAJOR_VERSION}_PRIVATE_API {" \
-                            "    qt_private_api_tag*;"
-
-        private_api_headers = $$SYNCQT.PRIVATE_HEADER_FILES $$SYNCQT.QPA_HEADER_FILES
-
-        for(header, private_api_headers): \
-            verscript_content += "    @FILE:$$header@"
-        verscript_content += "};"
-
-        current = Qt_$$QT_MAJOR_VERSION
-        verscript_content += "$$current { *; };"
-        isEmpty(QT_NAMESPACE): tag_symbol = qt_version_tag
-        else:                  tag_symbol = qt_version_tag_$$QT_NAMESPACE
-
-        for(i, 0..$$QT_MINOR_VERSION) {
-            previous = $$current
-            current = Qt_$${QT_MAJOR_VERSION}.$$i
-            equals(i, $$QT_MINOR_VERSION): verscript_content += "$$current { $$tag_symbol; } $$previous;"
-            else:                          verscript_content += "$$current {} $$previous;"
-        }
-
-        # Add a post-processing step to replace the @FILE:filename@
-        verscript_in = $${verscript}.in
-        verscriptprocess.name = linker version script ${QMAKE_FILE_BASE}
-        verscriptprocess.input = verscript_in
-        verscriptprocess.CONFIG += no_link target_predeps
-        verscriptprocess.depends = $$private_api_headers
-        verscriptprocess.output = $$verscript
-        verscriptprocess.commands = perl $${PWD}/data/unix/findclasslist.pl < ${QMAKE_FILE_IN} > $@
-        silent:verscriptprocess.commands = @echo creating linker version script ${QMAKE_FILE_BASE} && $$verscriptprocess.commands
-        QMAKE_EXTRA_COMPILERS += verscriptprocess
-
-        verscript = $$verscript_in
-    }
-    write_file($$OUT_PWD/$$verscript, verscript_content)|error()
-    unset(current)
-    unset(previous)
-    unset(verscript)
-    unset(verscript_content)
-}
-
 #install directives
 load(qt_installs)

diff --git a/qtbase/src/corelib/global/qversiontagging.cpp b/qtbase/src/corelib/global/qversiontagging.cpp
index b5e524bf4c..2678c57fb5 100644
--- a/qtbase/src/corelib/global/qversiontagging.cpp
+++ b/qtbase/src/corelib/global/qversiontagging.cpp
@@ -52,6 +52,7 @@
 #endif
 #define make_versioned_symbol(sym, m, n, separator)    make_versioned_symbol2(sym, m, n, separator)

+/*
 extern "C" {
 #if QT_VERSION_MINOR > 0
 make_versioned_symbol(SYM, QT_VERSION_MAJOR, 0, "@");
@@ -109,3 +110,4 @@ make_versioned_symbol(SYM, QT_VERSION_MAJOR, 15, "@");
 // the default version:
 make_versioned_symbol(SYM, QT_VERSION_MAJOR, QT_VERSION_MINOR, "@@");
 }
+*/
diff --git a/qtbase/src/corelib/global/qversiontagging.h b/qtbase/src/corelib/global/qversiontagging.h
index 75c2e9df7e..533a8e074a 100644
--- a/qtbase/src/corelib/global/qversiontagging.h
+++ b/qtbase/src/corelib/global/qversiontagging.h
@@ -81,9 +81,11 @@ QT_BEGIN_NAMESPACE
 #  endif
 #endif

+/*
 #if defined(QT_VERSION_TAG)
 QT_VERSION_TAG(qt_version_tag);
 #endif
+*/

 QT_END_NAMESPACE
