Submodule qtbase 8c8fba30..03439871:
diff --git a/qtbase/mkspecs/features/qt.prf b/qtbase/mkspecs/features/qt.prf
index 89f4946c50..3edf890da0 100644
--- a/qtbase/mkspecs/features/qt.prf
+++ b/qtbase/mkspecs/features/qt.prf
@@ -244,17 +244,6 @@ for(ever) {
     message("This is not a bug, but a result of using Qt internals. You have been warned!")
 }

-!no_qt_rpath:!static:qtConfig(rpath):!qtConfig(static):\
-        contains(all_qt_module_deps, core) {
-    relative_qt_rpath:!isEmpty(QMAKE_REL_RPATH_BASE):contains(INSTALLS, target):\
-            isEmpty(target.files):isEmpty(target.commands):isEmpty(target.extra) {
-        # NOT the /dev property, as INSTALLS use host paths
-        QMAKE_RPATHDIR += $$relative_path($$[QT_INSTALL_LIBS], $$qtRelativeRPathBase())
-    } else {
-        QMAKE_RPATHDIR += $$[QT_INSTALL_LIBS/dev]
-    }
-}
-
 !isEmpty(QMAKE_LFLAGS_RPATHLINK):!qtConfig(static) {
     # -rpath-link is used by the linker to find dependencies of dynamic
     # libraries which were NOT specified on the command line.
