Submodule qtbase 03439871..2e5ce799:
diff --git a/qtbase/src/corelib/tools/qcollator_posix.cpp b/qtbase/src/corelib/tools/qcollator_posix.cpp
index 81f97a02e1..6289e41c1c 100644
--- a/qtbase/src/corelib/tools/qcollator_posix.cpp
+++ b/qtbase/src/corelib/tools/qcollator_posix.cpp
@@ -51,11 +51,7 @@ void QCollatorPrivate::init()
     if (!isC()) {
         if (locale != QLocale())
             qWarning("Only C and default locale supported with the posix collation implementation");
-        if (caseSensitivity != Qt::CaseSensitive)
-            qWarning("Case insensitive sorting unsupported in the posix collation implementation");
     }
-    if (numericMode)
-        qWarning("Numeric mode unsupported in the posix collation implementation");
     if (ignorePunctuation)
         qWarning("Ignoring punctuation unsupported in the posix collation implementation");
     dirty = false;
