Submodule src/3rdparty 91432e048..1740e89b1:
diff --git a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/event_converter_evdev_impl.cc b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/event_converter_evdev_impl.cc
index eb8923faf28..2e7dfc9586b 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/event_converter_evdev_impl.cc
+++ b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/event_converter_evdev_impl.cc
@@ -8,6 +8,10 @@
 #include <linux/input.h>
 #include <stddef.h>

+#ifndef SYN_DROPPED
+#define SYN_DROPPED 3
+#endif
+
 #include "base/trace_event/trace_event.h"
 #include "ui/events/devices/stylus_state.h"
 #include "ui/events/event.h"
diff --git a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/gamepad_event_converter_evdev.cc b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/gamepad_event_converter_evdev.cc
index 3aec220b996..efa72887c31 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/gamepad_event_converter_evdev.cc
+++ b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/gamepad_event_converter_evdev.cc
@@ -8,6 +8,10 @@
 #include <linux/input.h>
 #include <stddef.h>

+#ifndef SYN_DROPPED
+#define SYN_DROPPED 3
+#endif
+
 #include "base/trace_event/trace_event.h"
 #include "ui/events/event_utils.h"
 #include "ui/events/ozone/evdev/device_event_dispatcher_evdev.h"
diff --git a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_event_converter_evdev.h b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_event_converter_evdev.h
index 2bbdc40326c..235f2f4e4f1 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_event_converter_evdev.h
+++ b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/evdev/touch_event_converter_evdev.h
@@ -18,6 +18,10 @@
 #define MT_TOOL_PALM 2
 #endif

+#ifndef SYN_DROPPED // < v2.6.39 kernel headers
+#define SYN_DROPPED 3
+#endif
+
 #include "base/compiler_specific.h"
 #include "base/files/file_path.h"
 #include "base/files/scoped_file.h"
diff --git a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/gamepad/static_gamepad_mapping.cc b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/gamepad/static_gamepad_mapping.cc
index 79f699a1f2d..f3acc60e096 100644
--- a/qtwebengine/src/3rdparty/chromium/ui/events/ozone/gamepad/static_gamepad_mapping.cc
+++ b/qtwebengine/src/3rdparty/chromium/ui/events/ozone/gamepad/static_gamepad_mapping.cc
@@ -8,6 +8,22 @@
 #include <map>
 #include <vector>

+#ifndef BTN_TRIGGER_HAPPY1
+#define BTN_TRIGGER_HAPPY1 0x2c0
+#endif
+#ifndef BTN_TRIGGER_HAPPY2
+#define BTN_TRIGGER_HAPPY2 0x2c1
+#endif
+#ifndef BTN_TRIGGER_HAPPY3
+#define BTN_TRIGGER_HAPPY3 0x2c2
+#endif
+#ifndef BTN_TRIGGER_HAPPY4
+#define BTN_TRIGGER_HAPPY4 0x2c3
+#endif
+#ifndef BTN_TRIGGER_HAPPY17
+#define BTN_TRIGGER_HAPPY17 0x2d0
+#endif
+
 #include "base/macros.h"
 #include "ui/events/ozone/evdev/event_device_info.h"
 #include "ui/events/ozone/gamepad/static_gamepad_mapping.h"
