Submodule qtwebengine f2cd5c1a..33ead218:
Submodule src/3rdparty 493af3794..2792bf94b:
diff --git a/qtwebengine/src/3rdparty/chromium/net/socket/udp_socket_posix.h b/qtwebengine/src/3rdparty/chromium/net/socket/udp_socket_posix.h
index c82f11b4535..c2768937062 100644
--- a/qtwebengine/src/3rdparty/chromium/net/socket/udp_socket_posix.h
+++ b/qtwebengine/src/3rdparty/chromium/net/socket/udp_socket_posix.h
@@ -32,9 +32,9 @@
 #include "net/traffic_annotation/network_traffic_annotation.h"
 
 #if defined(__ANDROID__) && defined(__aarch64__)
-#define HAVE_SENDMMSG 1
+#define HAVE_SENDMMSG 0
 #elif defined(OS_LINUX)
-#define HAVE_SENDMMSG 1
+#define HAVE_SENDMMSG 0
 #else
 #define HAVE_SENDMMSG 0
 #endif
